Formula/burst.rb

```diff
@@ -0,0 +1,55 @@
+class Burst < Formula
+  desc "Radix sort, lazy ranges and iterators, and more. Boost-like header-only library"
+  homepage "https://github.com/izvolov/burst"
+  url "https://github.com/izvolov/burst/archive/refs/tags/v3.1.0.tar.gz"
+  sha256 "12089244b7dbb26c3198f2ea5b80555050c38efe0a33ce4de4c1fff9b1e7532d"
+  license "BSL-1.0"
+  head "https://github.com/izvolov/burst.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "doctest" => :build
+  depends_on "boost"
+
+  def install
+    cmake_args = std_cmake_args + ["-DCMAKE_BUILD_TYPE=Release"]
+    system "cmake", "-S", ".", "-B", "build", *cmake_args
+    system "cmake", "--build", "build", "--target", "check"
```

Comment:
```suggestion
    system "cmake", "--build", "build"
```
We don't run `check` targets. They often either take too long, or fail for spurious reasons, or both. (We might even have the latter here.)

Reply:
In my case `build` target is needed only for running tests, because library is header-only.
Should I throw it away and run `install` immediately?

Tests run very quicky, btw:
```
$ time make check
[100%] Built target burst-unit-tests
[doctest] doctest version is "2.4.9"
[doctest] run with "--help" for options
===============================================================================
[doctest] test cases:  724 |  724 passed | 0 failed | 0 skipped
[doctest] assertions: 1036 | 1036 passed | 0 failed |
[doctest] Status: SUCCESS!
[100%] Built target check

real	0m0.976s
user	0m0.537s
sys	0m0.218s
```